src/root/DepositManager.ts

```diff
@@ -17,6 +20,14 @@ export default class DepositManager extends ContractsBase {
     )
   }
 
+  async depositDataByID(depositId: number, childChainAddress: address) {
```

Comment:
- `depositId:` will need to be an instance of big number or string.
- Probably shouldn't call it `depositData` because this function is simply returning a bool.